app/models/bookable_slot.rb

```diff
@@ -8,26 +8,26 @@ def self.regenerate_for_six_weeks
   end
 
   def self.within_date_range(from, to)
-    where('usable_slots.start_at > ? AND usable_slots.end_at < ?', from, to)
+    where('bookable_slots.start_at > ? AND bookable_slots.end_at < ?', from, to)
```

Comment:
Problem is, when combining this that `LEFT JOIN` further up we'll have problems. Feels safe to be explicit, or at least interpolate the table name in there.